test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -223,6 +223,7 @@ Function sys.bbf_varbinary_cmp(sys.bbf_varbinary,sys.bbf_varbinary)
 Function sys.bbfbinary_sqlvariant(sys.bbf_binary)
 Function sys.bbfvarbinary(sys.bbf_varbinary,integer,boolean)
 Function sys.bbfvarbinary_sqlvariant(sys.bbf_varbinary)
+Function sys.bigint_avg(internal)
```

Comment:
There are C type function (final functions) that are being used when the aggregate are called . There are dependency objects(functions,procedures,views) in the test files. do we need to have dependency tests for them explicitly?

Reply:
Talk to Rishabh, see if we are taking in account indirect dependencies or not?

Reply:
Currently we are not tracking indirect dependencies as even pg_depend does not track such dependencies. This looks like a function to function dependency where aggregate function depends upon other functions. But IMO, adding test for aggregate function should be enough since it's helper functions will anyway get executed/tested as part of it.